manufacturers/prob_match_manufacturers.R

```diff
@@ -0,0 +1,186 @@
+# Keying the drug company data
+# S. Kirmer
+# Feb 2017
+# Data4Democracy
+
+library(dplyr)
+library(data.table)
+library(reshape2)
+library(sem)
+library(doBy)
+library(ggplot2)
+library(AER)
+library(ivpack)
+library(readxl)
+library(RecordLinkage)
+library(stringr)
+library(gdata)
+library(stringdist)
+library(stats)
+
+companies_drugs <- read.csv("U://Medicaid_Drug/drugdata_clean.csv", stringsAsFactors = FALSE)
```

Comment:
FWIW, data.world makes it unnecessary to download then read- it has a single link that you can just put in the read.csv() statement. :)